content/en/blog/_posts/2020-09-03-warnings/index.md

```diff
@@ -320,6 +320,19 @@ If you want to be sure you notice deprecations as soon as possible and get a jum
 This could be used in a CI job to apply manifests to a current server,
 and required to pass with a zero exit code in order for the CI job to succeed.
 
+## Validating Admission Policy
+
+_Added in 2023_
+
+We introduced [Validating Admission Policy](/docs/reference/access-authn-authz/validating-admission-policy) in 1.26 as an alpha feature.
+Starting in Kubernetes v1.27, a field `validationAction` is added in ValidatingAdmissionPolicyBinding which is used to declare
+how Validations of the referenced ValidatingAdmissionPolicy are enforced. When it sets to `Warn`, a validation failure is reported to the request client
+in HTTP Warning headers, with a warning code of 299.
+
+<img alt="kubectl applying a manifest file when there is an existing a ValidatingAdmissionPolicy and a validationAction set to Warn, displaying a warning message and exiting with a non-zero exit code."
+     src="kubectl-warnings-vap.png"
+     style="width:637px;max-width:100%;">
```

Comment:
```suggestion
<img alt="kubectl applying a manifest file when there is an existing a ValidatingAdmissionPolicy and a validationAction set to Warn, displaying a warning message and exiting with a non-zero exit code."
     src="kubectl-warning-vap.png"
     style="width:637px;max-width:100%;">
```

Try this. Or, rename the image.

Reply:
Thank you so much for catching this! I have updated. :)